builtin/builtin_manager.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Contributors to the Veraison project.
```

Comment:
Could you please also add a README.md to package builtin?
This way it is easy for reader to connect the dots with the rest of the sub-system